webapp/sources/rudder/rudder-web/src/main/elm/build-app.sh

```diff
@@ -21,6 +21,6 @@ PROJECTS=("Notifications" "Healthcheck" "Editor" "Onboarding" "Rules")
 for PROJECT in ${PROJECTS[*]}; do
   lower=$(echo "${PROJECT}" | tr '[:upper:]' '[:lower:]')
   cd ${ELM_DIR}/${lower}
-  elm-0.19.1 make --optimize sources/${PROJECT}.elm --output=generated/rudder-${lower}.js
+  elm-0.19.1 make  sources/${PROJECT}.elm --output=generated/rudder-${lower}.js
```

Comment:
is this change voluntary?

Reply:
not at all, pushed by mistake (but `--optimize` prevent from logging into browser console, not convenient for debugging...)

Reply:
isn't this the point of the build-dev.sh script?

Reply:
The `build-dev.sh` script call `build-app.sh` to compile elm files and then move compilated file into the right directory, but with `--optimize` option the compilation fails if we use logging lib in Elm files, so everytime we have to remove this option temporarily, but I often forget to put it back before push my changes (in fact I used `rudder-dev wip` and then `rudder-dev commit` so it pushed all changed files in my repo)